mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -1439,15 +1441,15 @@ msgstr "I have been observing you for a while now, I'm glad my followers managed
 "What was your name again? \n"
 "....  .....  ....\n"
 "${{name}}?\n"
-"That sounds like a good name for someone who illegally owns a Tuxemon.\n"
+"That sounds like a good name for someone who illegally owns a Tuxemon.\n" # Why would it be a 'good name' for this?
```

Comment:
Again, please don't add comments to the files like this.  If you have a question or suggestion about a specific dialog line, open the file in the GitHub file viewer, click the line number, then copy/paste the url (which will be linked to that line) and either bring it up in the discord or open a github issue and provide that link.  Its possible that these comments would be missed in code review, leading to "pollution" of comments.  Comments in a translation file should only provide context to the string.  Questions or suggestions don't belong.